packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -173,14 +173,14 @@ export abstract class TargetGroupBase extends cdk.Construct implements ITargetGr
   public healthCheck: HealthCheck;
 
   /**
-   * Default port configured for members of this target group
+   * List of constructs that need to be depended on to ensure the TargetGroup is associated to a load balancer
    */
-  protected readonly defaultPort: string;
+  public readonly loadBalancerConstructs = new Array<cdk.IConstruct>();
```

Comment:
Probably should prefix that name with an `_` since the user won't need to interact with this directly. Also, probably should be called something like `loadBalancerAttachments` or something.